docs/_news/news07.md

```diff
@@ -20,14 +20,16 @@ Curious? [Learn more](remoteoverview.html)!
 
 #### New Features and Highlights for 0.7.0
 
+**Monitoring Updates**
+- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Node.js, Liberty, and Spring projects. You can [enable or disable](https://github.com/eclipse/codewind/issues/1290) this feature with just a click.
+- Effortlessly [visualize data](https://github.com/eclipse/codewind/issues/977) like CPU usage, memory footprint, and web traffic on the included application monitoring dashboard at the click of a button. Codewind hosts the dashboard, so displaying the graphs won't eat into your project's resources. 
+
 **OpenShift Do (odo)**
 - You can now easily import existing projects into Codewind and [use odo](https://github.com/eclipse/codewind/issues/1115) to build and deploy projects to your OpenShift or OKD cluster.
-- Introducing [support for Java template](https://github.com/eclipse/codewind/issues/450) and Java projects can build and deploy [once configured](//permalink to landing page).
-- You no longer have to create your [docker registry secret](https://github.com/eclipse/codewind/issues/665) in Codewind.
-
-
+- Codewind now [supports Java templates](https://github.com/eclipse/codewind/issues/450), and Java projects can build and deploy after they have been [configured](mdt-che-odo-support.html).
+- You no longer have to create your [Docker registry secret](https://github.com/eclipse/codewind/issues/665) in Codewind.
```

Comment:
this isn't correct, the registry secret is just added through a Codewind extension UI instead of through the Che UI now.

Reply:
And the registry secret change isn't just for odo, it's for Codewind on Che in general (and for hybrid, but that's not a change, that's all new)

Reply:
How about having a Docker subheading (same level as odo and monitoring) with this change >
```
**Docker**

-  Your [Docker registry secret](https://github.com/eclipse/codewind/issues/665) is now dynamically created through a Codewind extension.

```